extensions/fairifier-rdf-extension/dtls/fairifier/VirtuosoResource.java

```diff
@@ -55,7 +55,8 @@ public void push() throws IOException, HttpException {
             this.out = this.getModelString();
             HttpResponse res = HttpUtils.put(this.host + this.directory + this.filename, this.out,
                     this.username, this.password);
-            if (res.getStatusLine().getStatusCode() < 200) {
+            if ((res.getStatusLine().getStatusCode() < 200)
```

Comment:
`< 200` implies that status codes in the 100 range are errors, which they are not (and i'm completely disregarding the webdav spec here). I'd say a proper check is `if > 400 then failed`.